taojieTan/assignments/cyclicSort/lc268/lc268.py

```diff
@@ -2,14 +2,39 @@
 
 # We are given an array containing ‘n’ distinct numbers taken from the range 0 to ‘n’. Since the array has only ‘n’ numbers out of the total ‘n+1’ numbers, find the missing number.
 
+# input: an unsorted list with integers ranging from 0 to n and length of n
+# output: an integer
+# edge case: empty list => return -1
+
+'''
```

Comment:
Nice high level overview! Could be helpful to type the solution twice to help with muscle memory.